packages/flutter_tools/lib/src/localizations/localizations_utils.dart

```diff
@@ -149,12 +147,12 @@ class LocaleInfo implements Comparable<LocaleInfo> {
 // See also //master/tools/gen_locale.dart in the engine repo.
 Map<String, List<String>> _parseSection(String section) {
   final Map<String, List<String>> result = <String, List<String>>{};
-  List<String> lastHeading;
+  List<String> lastHeading = <String>[];
```

Comment:
Looking at [the data](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/localizations/language_subtag_registry.dart), it looks safe to assume that this will be initialized before it's referenced.
```suggestion
  late List<String> lastHeading;
```